index.html

```diff
@@ -4,16 +4,196 @@
-    <!-- Add a link to your custom CSS here -->
-
+    <link rel="stylesheet" href="css/main.css">
     <title>Responsive grid project</title>
 </head>
 
 <body>
 
-    <!-- Add your markup here -->
+
+<header class="container-fluid">
+    <nav class="row navbar">
+        <div class="col-md-7 toptext"> 
+            <div class="wrapsquare">
+                <div class="square">   
+                </div>
+                <h3>Nigel Pelvin </h3>
+                <h3 class="developer"> / FULL STACK DEVELOPER</h3>
+                <img class="hamburger" src="img/Hamburger_icon.svg.png" alt="hamburger menu" onclick="myFunction()">
+            </div>
+        </div>
+
+        <div class="col-md-5"> 
+            <div class="row">
+                <div class="col-md-3 hidesmall hidex" id="myDIVx"><button class="button3" style="margin-left:90%"  onclick="myFunction2()">X</button>
+                </div>
+
+                <div class="col-md-3 hidesmall" id="myDIV">
+                    <a href="index.html" class="menuLink">ABOUT ME</a>
+                </div>
+
+                <div class="col-md-3 hidesmall" id="myDIV2">            
+                    <a href="resume.html" class="menuLink">RESUME</a>
+                </div>
+
+                <div class="col-md-3 hidesmall" id="myDIV3">            
+                    <a href="projects.html" class="menuLink">PROJECTS</a>
+                </div>
+
+                <div class="col-md-3 hidesmall" id="myDIV4">            
+                    <a href="contact.html" class="menuLink">CONTACT</a>
+                </div>
+            </div>
+        </div>
+    </nav>
+</header>
+
+
+<div class="container main floatwrapper">
+    <div class="row">
+        <div class="col-md-7"> 
+
+
+
+<section class="float">
+    <div class="row">
+        <img src="img/profile.jpg" alt="Nigel Pelvin" class="roundimage">
+    </div>
+
+    <div class="row">
+        <h3>Nigel</h3>
+        <h3>Pelvin</h3>
+        <div class="line"></div>
+    </div>
+
+     <div class="row buttonrow">        
+         <div class="col">            
+            <a href="resume.html"><button type="button" class="button1">RESUME</button></a>
+        </div>
+        <div class="col">            
+            <a href="projects.html"><button type="button" class="button2">PROJECTS</button></a>
+        </div>
+    </div>
+
+     <div class="row"> 
+        <div class="col whitebg">
+            <a class="socials" href="#"><img src="img/iconmonstr-facebook-1.svg" alt="Twitter"></a>
+            <a class="socials" href="#"><img src="img/iconmonstr-twitter-1.svg" alt="Twitter"></a>
+            <a class="socials" href="#"><img src="img/iconmonstr-linkedin-1.svg" alt="LinkedIn"></a>
+            <a class="socials" href="#"><img src="img/iconmonstr-instagram-11.svg" alt="instagram"></a>
+
+        </div>
+    
+</section>
+</div>
+
+        <div class="col-md-5"> 
```

Comment:
On mobile this whole text section looks like it should not span the whole width, but perhaps be 10 columns, with an offset of 1.